E3Next/Processors/Movement.cs

```diff
@@ -58,42 +58,26 @@ public static void Check_Chase()
                 if (_chaseTarget != String.Empty && !Assist.IsAssisting)
                 {
                     double distance = MQ.Query<double>($"${{Spawn[={_chaseTarget}].Distance}}");
+                    double minDistanceToChase = E3.GeneralSettings.MinDistanceToChase;
```

Comment:
usually we have the variables also contain their section name in them

Reply:
Fixed, thanks for the callout